### PR TITLE
Bump operator version and add missing changelog entries.

### PR DESCRIPTION
**Pull request by porridge**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -53,12 +84,3 @@ and this project adheres to
...
 - Unpriviledged container execution
-
-[unreleased]:
-  https://github.com/mesosphere/kudo-cassandra-operator/compare/v3.11.5-0.1.2...HEAD
-[3.11.5-0.1.2]:
-  https://github.com/mesosphere/kudo-cassandra-operator/compare/v3.11.5-0.1.1...v3.11.5-0.1.2
-[3.11.5-0.1.1]:
-  https://github.com/mesosphere/kudo-cassandra-operator/compare/v3.11.4-0.1.0...v3.11.5-0.1.1
-[3.11.4-0.1.0]:
-  https://github.com/mesosphere/kudo-cassandra-operator/releases/tag/v3.11.4-0.1.0
```
Comment (mpereira) [inline CHANGELOG.md:64]: These lines provide the Markdown link URL which are referenced in the document.

Comment (mpereira) [reply CHANGELOG.md:64]: According to the https://keepachangelog.com/en/1.0.0/ "standard".

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,19 +6,50 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to
 [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
```
Comment (mpereira) [inline CHANGELOG.md:9]: If this file still follows the https://keepachangelog.com/en/1.0.0/ "standard" there should be a running "Unreleased" entry which is added to between releases.

File: CHANGELOG.md (hunk 1 above)
Comment (porridge) [reply CHANGELOG.md:64]: :man_facepalming: restored.

File: CHANGELOG.md (hunk 2 above)
Comment (porridge) [reply CHANGELOG.md:9]: Done.

File: metadata.sh [hunk 3]
```diff
@@ -19,7 +19,7 @@ export OPERATOR_NAME="cassandra"
 
 # More details about KUDO Versioning:
 # https://github.com/kudobuilder/kudo/pull/1028
-export OPERATOR_VERSION="0.1.2"
+export OPERATOR_VERSION="0.2.0"
```
Comment (zmalik) [inline metadata.sh:22]: let's bump this to `1.0.0` 
cc: @ANeumann82

Comment (porridge) [reply metadata.sh:22]: We should bump before the actual release. This is just for the preliminary one that I'm doing to test the release tooling.

**Comment by porridge** [comment]
@zmalik @ANeumann82 PTAL, I had to do some more changes for an effective version bump.

File: DEVELOPMENT.md [hunk 4]
```diff
@@ -359,7 +359,8 @@ This is achieved by creating and merging a PR _against the stable branch_ where:
 1.  (as needed) the various `*_VERSION` variables are set as necessary for base
     tech and the operator version, according to the versioning scheme shown
     above,
-1.  necessary files are updated by running `./tools/compile_template.sh`
+1.  necessary files are updated by running `./tools/compile_template.sh` and
+    `./tools/generate_parameters_markdown.sh`
```
Comment (ANeumann82) [inline DEVELOPMENT.md:363]:
```suggestion
1.  necessary files are updated by running 
    - `./tools/compile_template.sh` and
    - `./tools/generate_parameters_markdown.sh`
    - `./tools/format_files.sh`
```

Comment (porridge) [reply DEVELOPMENT.md:363]: I'll apply this separately in order not to delay this PR further.

Comment (porridge) [reply DEVELOPMENT.md:363]: Created PR https://github.com/mesosphere/kudo-cassandra-operator/pull/142

**Pull request merged**